config/config.go

```diff
@@ -363,8 +363,11 @@ func (c *Config) CreateAirbrakeNotifier() (*githubairbrake.Notifier, error) {
 		Timeout:   10 * time.Second,
 	}
 	errbitHost := c.Airbrake.Host
-	projectKey := c.Airbrake.ProjectKey
-	return githubairbrake.NewNotifierWithOptions(&githubairbrake.NotifierOptions{
+	projectKey, ok := os.LookupEnv("AIRBRAKE_PROJECT_KEY")
+	if !ok {
+		projectKey = c.Airbrake.ProjectKey
```

Comment:
Please add some logging here, it's always hard to debug these things without visibility. 
Something like:
* `using airbrake project key from AIRBRAKE_PROJECT_KEY env variable` (<- use const)
* `using airbrake project key from config file`

Reply:
updated with some modifications

Reply:
Sounds good. If we had the actual env variable name in the log that'd be helpful for debugging ("ahhhh I have a typo, I was setting `AIRBRAKE_API_KEY`, silly me!) but at least now we know which path it's taking.